roles/setup_assisted_installer/defaults/main.yml

```diff
@@ -85,3 +85,7 @@ assisted_installer_hardware_validation:
       ram_mib: 32768
       disk_size_gb: 120
       installation_disk_speed_threshold_ms: 10
+
+use_local_mirror_registry: "{{ setup_registry_service | default(False) }}"
+
+config_file_path: /tmp/wip/config
```

Comment:
Do we want the default to be in tmp folder? Should it be in another folder? Something like /opt

Reply:
I used `/tmp/wip/config` to match the setup registry code. It should only be required for pulling the containers then it wont be used any more. If we want to clean this up I would prefer to do both in a different PR

Reply:
I recommend we stop using /tmp and instead create a crucible/tmp folder or .crucible folder in the home directory.

Reply:
This is truly a tmp directory though. XDG_RUNTIME_DIR is pointed at it during the pulling of the images as it contains the containers auth file.